projects/samples/environments/indoor/worlds/break_room.wbt

```diff
@@ -0,0 +1,787 @@
+#VRML_SIM R2020a utf8
+WorldInfo {
+  info [
+    "Simulation of the Festo's Robotino 3 robot."
+    "This model includes an accurate and fast simulation of the mecanum wheels."
+    "The robot can be equiped with various lidar sensors and cameras."
+    "The simulation shows a Robotino 3 moving in an industrial environment using a Braitenberg algorithm using the information received by its nine infrared sensors."
+    "It shows a conveyor platform concept too."
+  ]
+  title "Festo's Robotino 3 robot"
+  contactProperties [
+    ContactProperties {
+      material1 "WheelMat"
+      coulombFriction [
+        1.8, 0, 0.2
+      ]
+      frictionRotation 0.9648 0
+      bounce 0
+      forceDependentSlip [
+        10, 0
+      ]
+    }
+  ]
+}
+Viewpoint {
+  orientation -0.5820406895605077 -0.7607148250601696 -0.28730052319758187 0.9782687143123455
+  position -7.4212317453727 7.444678937109314 7.710251980441673
+}
+TexturedBackground {
+  texture "entrance_hall"
+  luminosity 0.5
+  skybox FALSE
+}
+TexturedBackgroundLight {
+  texture "entrance_hall"
+}
+Robotino3 {
+  translation -1.59 0 0
+  rotation -0.5773509358554485 0.5773509358554485 0.5773489358556708 2.0944
+  bodySlot [
+    Robotino3Platform {
+      extension [
+        ConveyorPlatform {
+        }
+      ]
+    }
+  ]
+}
```

Comment:
I would remove the robot from this example (we want to showcase the environment only.
(remove contact properties too).